fiboa_cli/convert.py

```diff
@@ -69,6 +76,15 @@ def list_all_converters(keys):
             pass
     return converters
 
-def read_converter(id):
-    module_name = f".datasets.{id}"
-    return importlib.import_module(module_name, package="fiboa_cli")
+
+def read_converter(_id):
+    module_name = f".datasets.{_id}"
+    module = importlib.import_module(module_name, package="fiboa_cli")
+    if not hasattr(module, "convert"):
+        try:
+            clazz = next(v for v in module.__dict__.values()
+                         if type(v) is type and issubclass(v, BaseConverter) and v != BaseConverter)
+            return clazz()
+        except StopIteration:
+            log("Missing convert function or Converter class for module {_id}", "warning")
+    return module
```

Comment:
I think we should add comments to the parts that can be refactored/simplified once we reimplemented all converters to the class based approach.

Reply:
Yes, will do